emcache/base.py

```diff
@@ -327,6 +327,25 @@ async def stats(self, memcached_host_address: MemcachedHostAddress, *args: str)
         Please see a lot of detailed information in the documentation.
         """
 
+    @abstractmethod
+    async def verbosity(
+        self,
+        memcached_host_address: Union[MemcachedHostAddress, MemcachedUnixSocketPath],
+        level: int,
+        *,
+        noreply: bool = False
+    ) -> None:
+        """Set verbosity level in runtime. Verbosity option print STDOUT/STDERR information server.
+        Increase level increase verbose information memcached.
+        1 - `print standard errors/warnings`
```

Comment:
we do not need to put these ones between ``.

Reply:
i remove ``